torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -24,21 +25,27 @@ def export_fx_to_onnx(
         fx_module_args: Sequence[Any],
     ) -> torch.onnx.ExportOutput:
         # Apply decomposition table to the input graph.
-        decomposed_module = passes.Decompose(
+        module = passes.Decompose(
             fx_module,
             self.decomposition_table,
             enable_dynamic_axes=self.options.dynamic_shapes,
         ).run(*fx_module_args)
 
+        # Resolve mutations.
```

Comment:
Can we have an explanation why we need to resolve mutations somewhere in the codebase? It's for future maintenance.

Reply:
There is a test case added. It fails otherwise.

Reply:
I misunderstood your comment... Sure.

Reply:
```
        # ONNX does not support views and mutations.
        # Functionalize to get a semantically equivalent graph without mutations.
```
wdyt?

Reply:
Thanks!